.github/workflows/standalone.yml

```diff
@@ -0,0 +1,429 @@
+name: Build & Release (neo-cli)
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        type: string
+        description: 'Version Number (Example: 1.0.0)'
+        required: true
```

Comment:
Just a suggestion: don't we want to automatically run this job on GH release trigger?

Reply:
I understand, but how about moving from manual trigger to automatical? Like this:
```
on:
  release:
    types:
      - published
```

Reply:
Wouldn't that trigger after a release though? Workflow goes like this.
1. Build Binaries & Cache binaries
2. Get cache & Build Debian packages & Cache again
3. Get cache & Publish Release with binaries
4. Clean-up Caches

Reply:
And that's the way how it works in real life:
We create and publish new release, it has only source code attached to it:
![image](https://github.com/neo-project/neo/assets/22775709/7b9eca35-fce8-41e2-91ba-9217455bf648)
After that our Build workflow is triggered by "release" event:
![image](https://github.com/neo-project/neo/assets/22775709/b42176ab-c379-445e-a0e9-8a3354038918)
Every building job of Build workflow uploads artifacts to the storage:
![image](https://github.com/neo-project/neo/assets/22775709/96855814-a088-426e-a33c-8cb65785c0a8)
And in the end every building job attaches its artifacts to the release:
![image](https://github.com/neo-project/neo/assets/22775709/6c388829-a796-489f-9145-d83813ccff89)
Also Docker images are being automatically uploaded to our Dockerhub, but it's not related to the topic.

Reply:
Docker Image @shargon !!! Lets do it.

Reply:
I'll read more into this when i have time. But i thought you should know, we can't use artifacts. We eat up space so fast and only have something like 500MB for that. Our space is very limited.

Reply:
Guys... we need the binaries/artifacts to be uploaded somewhere. This would enable people to just download and use it. Also, for `brew`, relying on binaries/artifacts with an SHA-256 file is the best fit.

If we're so miserable for not being able to pay more than 500Mb of storage in GitHub, my suggestion is: create an Amazon S3 bucket (or, in Digital Ocean) and add it there. Wdyt?

Reply:
https://github.com/nspcc-dev/gh-push-to-neofs/

Can even use some static nuget feed generator to have proper feeds.

Reply:
@shargon need you here

Reply:
> If we're so miserable for not being able to pay more than 500Mb of storage in GitHub

Committee members should be able to aford it, core devs are the last link in neo economics, 
I removed a lot of artifacts, and old workflows and still without free space, I don't know where is gone this space...